### PR TITLE
Es lint & githooks

### PR DESCRIPTION
**Pull request by gnomex909**
Added some fixes, to make everything work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev:frontend": "npx webpack-dev-server",
     "dev:backend": "node -r ts-node/register backend/index.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\""
```
Comment (EvidentlyCube) [inline package.json]: Just remove this for now

File: package.json [hunk 2]
```diff
@@ -20,13 +20,17 @@
...
     "hooks": {
-      "pre-commit": "lint-staged && npm run test"
+      "pre-commit": "lint-staged"
     }
   },
   "lint-staged": {
     "*.ts": [
       "eslint --fix",
       "git add"
+    ],
```
Comment (EvidentlyCube) [inline package.json]: I checked and `lint-staged` [uses  `micromatch`](https://github.com/okonet/lint-staged#filtering-files) library for filtering files.

`micromatch` [supports `?`](https://github.com/micromatch/micromatch#why-use-micromatch), so you can have only one configuration key here, `"*.tsx?"` and it should work. 

Also what about that `git add`? If you remove it does it stop the warning that you talked about?

Comment (gnomex909) [reply package.json]: After checking out what `git add` does here, according to [this guide](https://www.robertcooper.me/using-eslint-and-prettier-in-a-typescript-project), it looks like it's better to leave it, and simply ignore the warning.

And using `*.tsx?` returned no files that match, but `*.{ts,tsx}` worked

File: package.json (hunk 1 above)
Comment (gnomex909) [reply package.json]: I removed "test" script for now

File: package.json (hunk 2 above)
Comment (EvidentlyCube) [reply package.json]: `*.{ts,tsx}` is even better, much more clearer to a quick glance

**Pull request merged**